qt_gui/src/qt_gui/main_window.py

```diff
@@ -40,15 +41,42 @@ class MainWindow(DockableMainWindow):
 
     save_settings_before_close_signal = Signal(Settings, Settings)
 
-    def __init__(self):
+    def __init__(self, help_text=None):
         super(MainWindow, self).__init__()
         self.setObjectName('MainWindow')
+        self._help_widget = None
+
+        if help_text:
+            font = QFont()
+            font.setPointSize(14)
+
+            self._help_widget = QTextBrowser(self)
+            self._help_widget.setFont(font)
+            self._help_widget.setReadOnly(True)
+            self._help_widget.setTextInteractionFlags(Qt.TextBrowserInteraction)
+            self._help_widget.setOpenExternalLinks(True)
+            self._help_widget.setHtml(help_text)
 
         self._save_on_close_signaled = False
         self._global_settings = None
         self._perspective_settings = None
         self._settings = None
 
+    def showHelpWidget(self, should_show):
+        if self._help_widget:
+            self._help_widget.show() if should_show else self._help_widget.hide()
+
+    def resizeEvent(self, event):
+        size = self.size()
+        info_percentage = 0.75
+        margin_percentage = (1.0 - info_percentage) / 2
+        if self._help_widget:
+            self._help_widget.setGeometry(
+                QRect(size.width() * margin_percentage,
+                      size.height() * margin_percentage,
+                      size.width() * info_percentage,
+                      size.height() * info_percentage))
```

Comment:
When trying this out, I had to cast each of these to `int`, like the following:

```suggestion
                QRect(int(size.width() * margin_percentage),
                      int(size.height() * margin_percentage),
                      int(size.width() * info_percentage),
                      int(size.height() * info_percentage)))
```

(otherwise I got a stack trace)